rasa_sdk/knowledge_base/storage.py

```diff
@@ -110,6 +110,14 @@ async def get_object(
         """
         raise NotImplementedError("Method is not implemented.")
 
+    async def get_object_types(self) -> List[Text]:
```

Comment:
Why is this method `async`? Looking at the implementation of this method in `InMemoryKnowledgeBase`, it doesn't make use of async programming.

Reply:
We have now removed this method from the KnowledgeBase class. We have only added the get_object_types() function in the InMemoryKnowledgeBase class as a normal function without async.